ark/docs/src/content/docs/objects.mdx

```diff
@@ -512,7 +512,7 @@ const myTuple = type(["...", type.number.array(), type.boolean, type.string])
 <a name="dates/literals" />
 ##### literals
 
-Date literals represent a Date instance with an exact value.
+Date literals represent a Date instance with an exact value. Date literal string are not validated statically, just passed to Date constructor at runtime.
```

Comment:
I'm not sure this makes the most sense to include explicitly as is- it would not really be feasible to validate every possible JS date format even if we wanted to.

I do think it makes sense to provide a bit more info here. Maybe something like:

> The value of a Date literal is determined by constructing a `new Date(dateLiteralContents)`. A Date literal that results in an invalid Date will throw a `ParseError`.

Reply:
Sure, the point that I was missing/info that I was looking for was just that a date string can be any valid string that could be passed to `Date()` -- arktype has so much awesome string parsing magic elsewhere that I'd initially assumed it was doing some sort of static analysis/matching/validation of the date strings (and since, as you said, it would not really be feasible to validate every possible JS date format, I figured there was perhaps some more limited list of acceptable date formats)

Reply:
Yeah I do actually have functionality like that implemented that lets you specify a global type-level date format and then would enforce that.

Just haven't finished integrating it yet :P 

![image](https://github.com/user-attachments/assets/2f9678ad-7cce-4c20-b1bb-932381e81d82)

Reply:
ooh, that would be awesome!